pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -34,6 +34,8 @@ import (
 	"go.goms.io/fleet/pkg/utils/labels"
 )
 
+var resourceSnapshotSize = 1024 * (1 << 10) // 1 MB
```

Comment:
Add a comment to explain why we set the limit as 1MB and a comment to explain why we need this.
```suggestion
var resourceSnapshotResourceSizeLimit = 1024 * (1 << 10) // 1 MB
```